src/card_number/mod.rs

```diff
@@ -1,42 +1,46 @@
-pub fn verify_card_number<T: AsRef<str>>(number: T) -> bool {
```

Comment:
honestly, I think a more versatile api is to provide both a raw function to be used, and an extension trait as we do here.

Reply:
I still can call this by `CardVerification::is_valid_card_number(...)` or do you mean something else?

Reply:
I meant keeping the function and writing an extension trait around them, but not a big deal at this point :D